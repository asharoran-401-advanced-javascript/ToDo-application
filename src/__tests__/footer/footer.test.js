/* eslint-disable no-unused-vars */
// eslint-disable-next-line strict
'use strict';

import React from 'react';
import {shallow} from 'enzyme';
import renderer from 'react-test-renderer';
import Footer from '../../component/footer/footer.js';

describe( '<Footer />' , () => {
  it('Footer is exist as end Todo Application' , () => {
    let app = shallow(<Footer />);
    expect(app.find('footer').exists).toBeTruthy();
  });
  //renderer is a package thatâ€™s able to take a React component and render it as a pure JavaScript object.
  it('render correctly' , () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});