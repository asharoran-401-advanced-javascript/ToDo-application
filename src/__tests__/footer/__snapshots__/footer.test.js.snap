// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> render correctly 1`] = `
<footer>
  copy Right by As-har Oran
</footer>
`;
